dev/com.ibm.websphere.appserver.features/visibility/protected/io.openliberty.appserver.connectors-2.0.feature

```diff
@@ -13,3 +13,4 @@ IBM-API-Package: jakarta.resource; type="spec", \
 -bundles=io.openliberty.jakarta.connectors.2.0; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.resource:jakarta.resource-api:2.0.0"
 kind=beta
 edition=base
+WLP-Activation-Type: parallel
```

Comment:
Do you want to also update com.ibm.websphere.appserver.javax.connector-1.6.feature and 
com.ibm.websphere.appserver.javax.connector-1.7.feature?